tools/travis-setup-rmq.sh

```diff
@@ -1,6 +1,13 @@
 #!/usr/bin/env bash
 
+source tools/travis-common-vars.sh
+NAME=$(db_name rmq)
+RMQ_PORT=5672
+
+docker rm -v -f $NAME || echo "Skip removing the previous container"
 docker run -d \
-       --name mongooseim-rmq \
-       -p 5672:5672 \
+       --name $NAME \
+       -p $RMQ_PORT:$RMQ_PORT \
```

Comment:
Again, this must be:
```suggestion
       -p $RMQ_PORT:5672 \
```
I know the first port is the host port, that's exactly what I said, that's the one we want to have configurable. The container's port is fixed, we can't change that.